webpack/scenes/ContentViews/__tests__/contentViewList.fixtures.json

```diff
@@ -0,0 +1,228 @@
+{
+  "total": 2,
+  "subtotal": 2, "page": 1,
+  "per_page": 20,
+  "error": null,
+  "search": null,
+  "sort": {
+    "by": "name",
+    "order": "asc"
+  },
+  "results": [
+    {
+      "composite": false,
+      "component_ids": [],
+      "default": false,
+      "force_puppet_environment": false,
+      "version_count": 1,
+      "latest_version": "1.0",
+      "auto_publish": false,
+      "solve_dependencies": false,
+      "repository_ids": [
+        1
+      ],
+      "id": 2,
+      "name": "cv1",
+      "label": "cv1",
+      "description": null,
+      "organization_id": 1,
+      "organization": {
+        "name": "Default Organization",
+        "label": "Default_Organization",
+        "id": 1
+      },
+      "created_at": "2020-03-19 15:08:59 UTC",
+      "updated_at": "2020-03-19 15:09:11 UTC",
+      "environments": [
+        {
+          "id": 1,
+          "name": "Library",
+          "label": "Library",
+          "permissions": {
+            "readable": true
+          }
+        }
+      ],
+      "repositories": [
+        {
+          "id": 1,
+          "name": "zoo3",
+          "label": "zoo3",
+          "content_type": "yum"
+        }
+      ],
+      "puppet_modules": [],
+      "versions": [
+        {
+          "id": 2,
+          "version": "1.0",
+          "published": "2020-03-19 15:09:11 UTC",
+          "environment_ids": [
+            1
+          ]
+        }
+      ],
+      "components": [],
+      "content_view_components": [],
+      "activation_keys": [],
+      "next_version": "2.0",
+      "last_published": "2020-03-19 15:09:11 UTC",
+      "permissions": {
+        "view_content_views": true,
+        "edit_content_views": true,
+        "destroy_content_views": true,
+        "publish_content_views": true,
+        "promote_or_remove_content_views": true
+      }
+    },
+    {
+      "composite": true,
+      "component_ids": [
+        27,
+        28
+      ],
+      "default": false,
+      "force_puppet_environment": false,
+      "version_count": 0,
+      "latest_version": null,
+      "auto_publish": false,
+      "solve_dependencies": false,
+      "repository_ids": [
+        104,
+        105,
+        106,
+        107,
+        109,
+        108,
+        111,
+        110
```

Comment:
I think using 2 contents repos and ids would be more than enough as a fixture,
And reduce the size of the snaps too

Reply:
There are no snaps since this is using r-t-lib, is there a more specific concern? These are actual API responses, the file is only 5kb and is just used in testing.